src/ca/mcgill/cs/jetuml/viewers/edges/SegmentedEdgeViewer.java

```diff
@@ -110,6 +110,11 @@ private static void drawString(GraphicsContext pGraphics, Point2D pEndPoint1, Po
 			pGraphics.setTextBaseline(VPos.CENTER);
 			pGraphics.setTextAlign(TextAlignment.CENTER);
 		}
+		else
+		{
+			pGraphics.setTextBaseline(VPos.CENTER);
+			pGraphics.setTextAlign(TextAlignment.CENTER);
+		}

```

Comment:
This code can be moved outside the conditional statement, no?

Reply:
I changed the logic here slightly. See my response to your comment, but I added the alignments as parameters now